conference/src/main/java/com/kyriba/conference/management/domain/exception/EntityNotFound.java

```diff
@@ -0,0 +1,9 @@
+package com.kyriba.conference.management.domain.exception;
+
+public class EntityNotFound extends Exception
```

Comment:
All exceptions must have Exception suffix in the name and base exception should be RuntimeException because checked exception usage is an anti-pattern in the modern Java world

Reply:
fixed